api/database/functions.py

```diff
@@ -122,27 +196,30 @@ async def verify_token(token:str, verification:str, route:str=None) -> bool:
         Update the token's last-used field with the current time.
         Attempt the request.
     """
-    
+
     sql = select(ApiUser)
     sql = sql.where(ApiUser.token == token)
     sql = sql.where(ApiPermission.permission == verification)
     sql = sql.join(ApiUserPerm, ApiUser.id == ApiUserPerm.user_id)
-    sql = sql.join(ApiPermission, ApiUserPerm.permission_id == ApiPermission.id)
-    
+    sql = sql.join(ApiPermission, ApiUserPerm.permission_id ==
+                   ApiPermission.id)
+
     sql_usage = select(ApiUsage)
-    sql_usage = sql_usage.where(ApiUser.id == ApiUsage.user_id, ApiUser.token == token)
-    sql_usage = sql_usage.where(ApiUsage.timestamp >= datetime.utcnow() - timedelta(hours=1))
-    
+    sql_usage = sql_usage.where(
+        ApiUser.id == ApiUsage.user_id, ApiUser.token == token)
+    sql_usage = sql_usage.where(
+        ApiUsage.timestamp >= datetime.utcnow() - timedelta(hours=1))
+
```

Comment:
weird formatting

Reply:
This is what the linter created.

Reply:
I don't like it either but apparently it's pep8